app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<section class="login_section">
+  <h2>Log in</h2>
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field">
+      <%= f.label :email, class:"email_label"  %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Username" %>
```

Comment:
- [ ] This field placeholder indicates that the user should type their user name whereas it should be email. Kindly fix that.

  ![ieow](https://user-images.githubusercontent.com/81468675/216362883-eef7557d-c3a6-46c9-95e1-e0f71f30fa5b.jpg)

Reply:
@julie-ify, No problem!